src/stores/useMessages.ts

```diff
@@ -2,12 +2,18 @@ import type { MessagesState } from '@stores/types'
 import type { BotMessage, UserMessage } from '@models/Message'
 import { uniqueId } from 'lodash'
 import { useNotifications } from '@stores/useNotifications'
-import { apiClient, updateCredential } from '@/api'
 import MemoryService from '@services/MemoryService'
 import { useSettings } from './useSettings'
+import LogService from '@/services/LogService'
+import { useApiClient } from '../composables/useApiClient'
```

Comment:
Vite has the auto-imports plugin, no need for this explicit import of the composable.